src/main/webapp/app/iris/state-store.service.ts

```diff
@@ -227,11 +229,19 @@ export class IrisStateStore implements OnDestroy {
         }
         if (isRateLimitUpdatedAction(action)) {
             const castedAction = action as RateLimitUpdatedAction;
+            const rateLimitInfo = castedAction.rateLimitInfo;
+            let errorMessage: IrisErrorType | null = null;
```

Comment:
we prefer `undefined` over `null`

Reply:
Will be addressed in a separate PR, as this requires a bigger refactoring of the Iris client.